ios/Classes/HomeViewController.swift

```diff
@@ -87,13 +98,31 @@ class HomeViewController: UIViewController, ImageScannerControllerDelegate {
             window.rootViewController?.present(scanPhotoVC, animated: true)
         }
     }
+
+    @objc func showPhotos() {
```

Comment:
Here all the code is commented but the showPhotos is added as the button click handler.